kcbq-connector/quickstart/properties/connector.properties

```diff
@@ -22,8 +22,7 @@ sanitizeTopics=true
 
 autoUpdateSchemas=true
 
-schemaRetriever=com.wepay.kafka.connect.bigquery.schemaregistry.schemaretriever.SchemaRegistrySchemaRetriever
-schemaRegistryLocation=http://localhost:8081
+schemaRetriever=com.wepay.kafka.connect.bigquery.retrieve.IdentitySchemaRetriever
 
```

Comment:
Should we add the `RegexRouter` SMT here so users have an example of how to use it?

Reply:
Yes, I think so.